src/server/engine_factory.rs

```diff
@@ -185,11 +189,11 @@ impl KvEngineFactory {
         if let Some(filter) = self.create_raftstore_compaction_listener() {
             kv_db_opts.add_event_listener(filter);
         }
-        let _kv_cfs_opts = self.inner.rocksdb_config.build_cf_opts(
-            &self.inner.block_cache,
-            self.inner.region_info_accessor.as_ref(),
-            self.inner.api_version,
-        );
+        // let _kv_cfs_opts = self.inner.rocksdb_config.build_cf_opts(
```

Comment:
Don't comment it out, just update the `build_cf_opts` call.

Reply:
fixed.